renewer/huey.py

```diff
@@ -21,9 +21,7 @@
 huey = RedisHuey(connection_pool=connection_pool)
 
 # Normal task, no retries
-nonretriable_task = huey.context_task(db.session_handler(), as_argument=True)
+nonretriable_task = huey.task()
 
 # These tasks retry every 10 minutes for four hours.
-retriable_task = huey.context_task(
-    db.session_handler(), as_argument=True, retries=6 * 4, retry_delay=10 * 60
-)
+retriable_task = huey.task(retries=6 * 4, retry_delay=10 * 60)
```

Comment:
These both look fine, I'm just curious what the difference is between a `context_task` vs. just a `task` - in other words, what was the impetus behind the switch; no more need for the DB session to be passed due to the session handler context managers below?

Reply:
a [`context_task`](https://huey.readthedocs.io/en/latest/api.html#Huey.context_task) takes a context-manager object, calls `with ...:` on it, and passes it to the task as the first parameter. 
I think the problem I was running into was that I was using a contextmanager _function_ and not an object, so huey didn't like it.

I could probably convert `session_handler_ to be a class, then pass an object to Huey instead, but I think I'm fine just managing the sessions within the tasks instead for now.

Reply:
I replaced the `session_handler` function with a `SessionHandler` class, so `context_tasks` work now. I also added a tiny blurb on them where we define them